xml/System.Net.Http/HttpClient.xml

```diff
@@ -128,6 +120,9 @@ Connection pool properties can be configured on a <xref:System.Net.Http.HttpClie
 
 Disposing of the HttpClient instance closes the open connections and cancels any pending requests.
 
+> [!NOTE]
+> If you later send a request to the same server, a new connection is created. If the rate of requests is high, or if there are any firewall limitations, that can exhaust the available sockets because of default TCP cleanup timers.
```

Comment:
I am not sure this statement is correct. Maybe concurrent requests were meant? Because if you send the request *later*, the pooled connection will be reused...

Maybe this could be something like
```suggestion
> If you concurrently send HTTP/1.1 requests to the same server, new connections can be created. Even if you reuse `HttpClient` instance, if the rate of requests is high, or if there are any firewall limitations, that can exhaust the available sockets because of default TCP cleanup timers. To limit the number of concurrent connections, you can use `MaxConnectionsPerServer` property. By default, the number of concurrent HTTP/1.1 connections is unlimited.
```